tools/process-bounties/enricher.js

```diff
@@ -32,19 +32,25 @@ bounties.withPromise().then(async (bountyPaths) => {
     vulnerabilityDetails.Repository.Owner = repositoryUrlParts[3];
     vulnerabilityDetails.Repository.Name = repositoryUrlParts[4];
 
+    // TEMPORARY: populate historical bounties with ForkName, default: Repository.Name
+    // Can be removed after first run
+    vulnerabilityDetails.Repository.ForkName =
+      vulnerabilityDetails.Repository.Name;
```

Comment:
Presumably, we will manually update the others that include `...-1`?

Reply:
Correct, they can be referenced here: Historical exceptions: https://github.com/418sec?q=-1&type=&language=

Reply:
We should prepend this with `'418sec' +`?

Reply:
In https://github.com/418sec/huntr-ddb-importer/pull/3 we reference just the name of the Fork rather than `full_name`, let me know which you would prefer and I can append it to both if you like. Cheers 👻

Reply:
Apologies, misinterpreted this from another file:

`${repoOwner}/${repoName} > 418sec/${forkName}`

Reply:
Sorry, ignore that reference. Here was what I was meaning: https://github.com/418sec/huntr/blob/3a2c0e3db0b37755580d9bac8847fecab83e4ea4/tools/disclosure-accepted/create-fork.js#L24

Reply:
Thanks for clarifying with me over the phone! 🍰